docs/conf.py

```diff
@@ -208,12 +202,9 @@
 
 html_context = {"analytics_enabled": True}
 
-import sphinx
-
-if sphinx.__version__ >= "4.1.0":
```

Comment:
You pin Sphinx to >=5, so this is unnecessary.

Reply:
I imagine it was just overlooked when sphinx was recently pinned that way following Terra - since it makes no net difference to the outcome. But yes it can be dropped now.